driver-scala/src/integration/scala/org/mongodb/scala/RetryableReadsTest.scala

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright 2008-present MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.mongodb.scala
-
-import com.mongodb.client.AbstractRetryableReadsTest
-import org.bson.{ BsonArray, BsonDocument, BsonString }
-import org.mongodb.scala.syncadapter.SyncMongoClient
-
-class RetryableReadsTests(
```

Comment:
Is there a unified test for Scala now?

Reply:
No.  Given that Kotlin, Kotlin co-routine, and Scala are all thin wrappers around the Java-based drivers, I don't think it's worth it to test retryable reads against all of them, as all the logic that's being tested lives in driver-core and is well-tested by the test runners for sync and reactive drivers.

Comparing with the other unified tests, I see only UnifiedCrudTest for kotlin-coroutine and kotlin-sync, so this seems in line with previous decisions.